lib/foreman_maintain/concerns/metadata.rb

```diff
@@ -104,35 +138,15 @@ def metadata
       end
 
       def label
-        metadata[:label] || generate_label
+        self.class.label
```

Comment:
This is good :+1: 
`class.label` would have produced the same results right? Just clarifying myself. No need to change, I prefer `self.class` chaining for readability.